components/admin/requests/forms/DoctorInformationForm.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  Text,
+  Stack,
+  FormHelperText,
+  Box,
+  Divider,
+} from '@chakra-ui/react'; // Chakra UI
+import { DoctorInformation } from '@tools/components/admin/requests/forms/doctor-information-form';
+
+type DoctorInformationFormProps = {
+  readonly doctorInformation: DoctorInformation;
+  readonly onChange: (updatedData: DoctorInformation) => void;
+};
+/**
+ * Form component for doctor information with styling.
+ * @param props - Props
+ * @returns doctor information form.
+ */
+export default function DoctorInformationForm({
+  doctorInformation,
+  onChange,
+}: DoctorInformationFormProps) {
+  return (
+    <>
+      {/* Personal Information Section */}
+      <Box paddingBottom="32px">
+        <Stack direction="row" spacing="20px" paddingBottom="24px">
+          <FormControl isRequired>
+            <FormLabel>{'First name'}</FormLabel>
```

Comment:
Add a TODO comment for the first and last name fields mentioning that we need to separate the names in the future